pkg/pod/network.go

```diff
@@ -13,3 +13,14 @@ func StaticIPAnnotation(name string, ipAddr []string) []*multus.NetworkSelection
 		},
 	}
 }
+
+// StaticIPAnnotationWithInterfaceAndNamespace defines static ip address, interface name and namespace
+// network annotation for pod object.
+func StaticIPAnnotationWithInterfaceAndNamespace(
+	name, namespace, intName string, ipAddr []string) []*multus.NetworkSelectionElement {
+	baseAnnotation := StaticIPAnnotation(name, ipAddr)
```

Comment:
Should you check name, namespace, and intName are not "" ?

Reply:
Yes right. We need to add validation function here. I think it'll be my next PR because this change is going to affect both projects it would require multiple PRs in to diff project.